source/Line.cpp

```diff
@@ -531,12 +531,15 @@ Line::GetLineOutput(OutChanProps outChan)
 		return rd[outChan.NodeID][1];
 	else if (outChan.QType == VelZ)
 		return rd[outChan.NodeID][2];
-	else if (outChan.QType == Ten)
-		return getNodeTen(outChan.NodeID).norm();
+	else if (outChan.QType == Ten) {
+		if ((outChan.NodeID == 0) || (outChan.NodeID == N))
+			return getNodeForce(outChan.NodeID).norm();
+		return (0.5 * (T[outChan.NodeID] + T[outChan.NodeID - 1])).norm();
```

Comment:
I think here for internal nodes we should return the wall tension using `Line::getNodeTen(outChan.NodeID).norm()`. This would keep output channels consistent with what is done in MDF, where the subroutine GetLineTen is returned for the "Ten" output channel: https://github.com/OpenFAST/openfast/blob/6a7a543790f3cad4a65b87242a619ac5b34b4c0f/modules/moordyn/src/MoorDyn_IO.f90#L1885